modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -379,18 +379,23 @@ SHOW ALIAS `northwind-2022` FOR DATABASE YIELD name, properties
 [[alias-management-create-remote-database-alias]]
 === Create database aliases for remote databases
 
-A database alias can target a remote database by providing an URL and the credentials of a user on the remote Neo4j DBMS.
+A database alias can target a remote database by providing a URL and the credentials of a user on the remote Neo4j DBMS.
 See xref:database-administration/aliases/remote-database-alias-configuration.adoc[] for the necessary configurations.
 
 Since remote database aliases target databases that are not in this DBMS, they do not fetch the default Cypher version from their target like the local database aliases.
 Instead, they are assigned the version given by xref:configuration/configuration-settings.adoc#config_db.query.default_language[`db.query.default_language`], which is set in the `neo4j.conf` file.
 Alternatively, you can specify the version in the `CREATE ALIAS` or `ALTER ALIAS` commands.
 See xref:database-administration/aliases/manage-aliases-standard-databases.adoc#set-default-language-for-remote-database-aliases[] and xref:database-administration/aliases/manage-aliases-standard-databases.adoc#alter-default-language-remote-database-alias[] for more information.
 
+When creating a remote database alias, the credentials used to target to the remote DBMS need to be specified. This can be done by providing the credentials of a native user, or
+by forwarding the OIDC credentials of the user logged in to the local DBMS.
```

Comment:
We probably need to have one of these version tags somewhere indicating that CREDENTIAL FORWARDING is new.
<img width="195" height="49" alt="Bildschirmfoto 2025-10-31 um 09 20 23" src="https://github.com/user-attachments/assets/e22c16cf-c823-40f2-b6f0-23bf583ba7f6" />

I believe we would want that to show the version in which we set `internal.cypher.enable_oidc_credential_forwarding` to default to true. Before we set that flag to true I think we would want the SHOW updates to be done and also have done our manual testing.

Question for the docs reviewer: Do we ever document features that aren't released yet, e.g. "This feature is in beta and you need to set config x.y.z=true to access it"